src/onegov/fsi/security.py

```diff
@@ -0,0 +1,34 @@
+from onegov.core.security import Personal
+from onegov.fsi import FsiApp
+from onegov.fsi.models import CourseAttendee
+
+"""
+Since FSI is mainly for internal use, a user must logged in to see even
+courses.
+The standard has_permission_logged_in treats members almost like anon users
+
+The idea for permission is the following:
+
+Personal: beeing logged in by default, can be overwritten model wise
+Private: also editor can access it
+Secret: admins
+
+"""
+
+
+@FsiApp.permission_rule(model=object, permission=Personal)
+def local_is_logged_in(app, identity, model, permission):
+    return identity.role in ('admin', 'editor', 'member')
+
+
+@FsiApp.permission_rule(model=CourseAttendee, permission=object)
```

Comment:
Looks like this is still wrong, no?

Reply:
Sorry I requested a review for a past merge by accident